crates/optimism/chainspec/src/superchain/chain_specs.rs

```diff
@@ -51,3 +51,55 @@ create_chain_spec!("worldchain", "sepolia");
 create_chain_spec!("xterio-eth", "mainnet");
 create_chain_spec!("zora", "mainnet");
 create_chain_spec!("zora", "sepolia");
+create_superchain_enum!(
```

Comment:
great, 
I'm merging this as is, but we can even more simplify this if `create_chain_spec!` is invoked directly in the code that `create_superchain_enum!` generates then we don't need this twice, but we can do this separately